wallet/test/Test/Pos/Util/MnemonicSpec.hs

```diff
@@ -28,6 +28,7 @@ instance Eq CC.XPrv where
 
 type TestVector =
     ( BL.ByteString             -- ^ Raw JSON encoding (V1)
+    , BL.ByteString             -- ^ Precursor to legacy (V0) JSON encoding
```

Comment:
This is slightly awkward to describe. Could be more verbose if changing the comment format.

Reply:
As there is now no other comment in this file explaining the legacy format, this should probably be more detailed. Think I'll do it.